src/est/BUILD

```diff
@@ -0,0 +1,110 @@
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright (c) 2025, The OpenROAD Authors
+
+load("//bazel:tcl_encode_or.bzl", "tcl_encode")
+load("//bazel:tcl_wrap_cc.bzl", "tcl_wrap_cc")
+
+package(
+    default_visibility = ["//:__subpackages__"],
+    features = ["layering_check"],
+)
+
+cc_library(
+    name = "est",
+    srcs = [
+        "src/EstimateParasitics.cpp",
+        "src/MakeWireParasitics.cpp",
+        "src/MakeWireParasitics.h",
+        "src/OdbCallBack.cpp",
+        "src/SteinerRenderer.cpp",
+        "src/SteinerRenderer.h",
+        "src/SteinerTree.cpp",
+    ],
+    hdrs = [
+        "include/est/AbstractSteinerRenderer.h",
+        "include/est/EstimateParasitics.h",
+        "include/est/OdbCallBack.h",
+        "include/est/SteinerTree.h",
+    ],
+    includes = [
+        "include",
+    ],
+    deps = [
+        "//src/dbSta",
+        "//src/grt:types",
```

Comment:
I believe the purpose of grt:types was to workaround the circular dependency.  It should be possible to just depend on grt now and get rid of the :types target